### ISSUE TITLE
Add CPAN Required Lib

### ISSUE THREAD

**Issue by joserribeiro26** [RotherOSS/otobo#219]
After install, I accesss Admin Area e I need install this library

Unicode::Collate::Locale

My database default charset is utf8.
O.S: Centos 8
Database: MariaDB: 10.3

 Message: Can't locate Unicode/Collate/Locale.pm in @INC (you may need to install the Unicode::Collate::Locale module) (@INC contains: /opt/otobo/Custom /opt/otobo/Kernel/cpan-lib /opt/otobo/ /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 /etc/httpd) at /opt/otobo/Kernel/Output/HTML/NavBar/AdminFavourites.pm line 25.
BEGIN failed--compilation aborted at /opt/otobo/Kernel/Output/HTML/NavBar/AdminFavourites.pm line 25.
Compilation failed in require at /opt/otobo/Kernel/System/Main.pm line 93.

**Comment by bschmalhofer** [comment]
Obrigado José,

this is the first non-core contribution in this repository!

The issue is a bit strange. `Unicode::Collate::Local` has been a core Perl module since Perl 5.13.4.

```
corelist Unicode::Collate::Locale

Data for 2019-05-22
Unicode::Collate::Locale was first released with perl v5.13.4

```
But it looks like CentOS has messed with the list of included modules and provides Unicode::Collate::Locale in a separate rpm, https://www.rpmfind.net/linux/rpm2html/search.php?query=perl%28Unicode%3A%3ACollate%3A%3ALocale%29&submit=Search+...&system=centos&arch=

I'll add the module to the dependency list.

**Comment by bschmalhofer** [comment]
PR is applied. Closing.
